### ISSUE TITLE
Focus appearance / text contrast of footer links

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_microsites_base#307]
The default focus appearance of links in the footer is white text with a yellow background, which has insufficient contrast. 

Tested with the localgov_microsites_base theme with no overrides. 
Also a problem on our demo sites

- https://site3.demo.microsites.localgovdrupal.org/
- https://site1.demo.microsites.localgovdrupal.org/
